GooglePlacesAutocomplete.js

```diff
@@ -556,18 +606,33 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
         setStateText(props.preProcess(text));
       }
 
-      request.open(
-        'GET',
-        `${url}/place/autocomplete/json?input=` +
-          encodeURIComponent(text) +
-          '&' +
-          Qs.stringify(props.query),
-      );
+      if (props.isNewPlacesAPI) {
+        request.open('POST', `${url}/v1/places:autocomplete`);
+      } else {
+        request.open(
+          'GET',
+          `${url}/place/autocomplete/json?input=` +
+            encodeURIComponent(text) +
+            '&' +
+            Qs.stringify(props.query),
+        );
+      }
 
       request.withCredentials = requestShouldUseWithCredentials();
       setRequestHeaders(request, getRequestHeaders(props.requestUrl));
 
-      request.send();
+      if (props.isNewPlacesAPI) {
+        const { locationbias, locationBias, types, ...rest } = props.query;
+        request.send(
+          Qs.stringify({
```

Comment:
With the new back-end change I'm making, you should JSON-encode the body, and you may include location bias. You do need to strip out `query.key`, as it is not valid to pass it in the request body. See comment above about including it as a URL param.

```suggestion
        const { key, types, ...rest } = props.query;
        request.send(
          JSON.stringify({
```

Reply:
> With the new back-end change I'm making, you should JSON-encode the body, and you may include location bias. You do need to strip out `query.key`, as it is not valid to pass it in the request body.

1. I have made changes to use `JSON.stringify`
2. I have stripped out `query.key` 
3. For new places API, we use `locationBias`. So, let will strip `locationbias` which was part of old Places API.